src/unitycatalog/ai/databricks.py

```diff
@@ -408,17 +408,24 @@ def _execute_uc_function(
             if response.status and job_pending(response.status.state) and response.statement_id:
                 statement_id = response.statement_id
                 _logger.info("Retrying to get statement execution status...")
-                max_retries = 6
-                for retry_cnt in range(1, max_retries + 1):
-                    time.sleep(2**retry_cnt)
+                wait_time = 0
+                retry_cnt = 0
+                client_execution_timeout = int(os.environ.get("CLIENT_EXECUTION_TIMEOUT", "120"))
```

Comment:
```suggestion
                client_execution_timeout = int(os.environ.get("UNITICATALOG_AI_CLIENT_EXECUTION_TIMEOUT", "120"))
```

can we add a prefix?